packages/dynamic-import/client.js

```diff
@@ -127,7 +135,21 @@ function fetchMissing(missingTree) {
   // preflight OPTIONS request, which may add latency to the first dynamic
   // import() request, so it's a good idea for ROOT_URL to match
   // location.host if possible, though not strictly necessary.
-  var url = Meteor.absoluteUrl(fetchURL);
+
+  let url = fetchURL;
+
+  // Lo and behold!
+  const useDynamicOrigin = Meteor.settings
+      && Meteor.settings.public
+      && Meteor.settings.public.packages
+      && Meteor.settings.public.packages.dynamicImport
+      && Meteor.settings.public.packages.dynamicImport.useDynamicOrigin;
```

Comment:
You can use optional chaining, it’s built in feature for meteor

Reply:
@afrokick yeah I tried that but then I got some compile errors :disappointed:

Reply:
I had some time to take a look at this @afrokick and this is the error I get when I use optional changing...

![image](https://user-images.githubusercontent.com/17411182/86537339-70928100-bebc-11ea-80ce-4c60ea6ea462.png)

cc @filipenevola

I will try to find a workaround for learning purposes, but using the classical syntax should be fine as well.

Reply:
Apparently the `ecmascript` package itself depends on the `dynamic-import` package, which means that it cannot support newer ECMAScript code as it is part of the parsing implementation itself.

https://github.com/meteor/meteor/pull/9384

If I try to remove these lines I can use the `ecmascript` package (eliminating the circular dependency error) but not sure if it will break something else:

https://github.com/meteor/meteor/pull/9384/files#diff-adfc7aa5c6b9d8ed73279b06d8873386R25-R27

Reply:
Do note that this also means you should not use const & let etc.